development/.env.example

```diff
@@ -0,0 +1 @@
+PYTHON_VER=3.11
```

Comment:
```suggestion
# Required for vscode Dev Containers support
PYTHON_VER=3.11
```

I tested using the `invoke build`/`invoke debug` container workflow while a `development/.env` file was present and this file does not seem to negatively impact that workflow.

Reply:
I'm curious, where do you put this variable usually?

Reply:
The default is set in tasks.py but can be overridden in `invoke.yml`

Reply:
Ok, I see. But it won't work with the devcontainer workflow. I tried setting the var in `remoteEnv` and `containerEnv`, but unfortunately it doesn't work.

Reply:
Yeah I tried that too. I just wanted to make sure that this file (`development/.env`) wasn't going to break the other development workflows.